cmd/infracost/breakdown.go

```diff
@@ -50,6 +50,14 @@ func breakdownCmd(ctx *config.RunContext) *cobra.Command {
 	cmd.Flags().String("format", "table", "Output format: json, table, html")
 	cmd.Flags().StringSlice("fields", []string{"monthlyQuantity", "unit", "monthlyCost"}, "Comma separated list of output fields: all,price,monthlyQuantity,unit,hourlyCost,monthlyCost.\nSupported by table and html output formats")
 
+	// Hidden experimental flags
+	cmd.Flags().Bool("terraform-parse-hcl", false, "Parse the HCL directly instead of generating a Terraform plan. This option does not need credentials and is faster (experimental)")
+	cmd.Flags().StringSlice("terraform-var-file", nil, "Load variable files from the given file, similar to Terraform's -var-file flag. Only supported with --terraform-parse-hcl (experimental)")
+	cmd.Flags().StringSlice("terraform-var", nil, "Set a value for one of the input variables, similar to Terraform's -var flag. Only supported with --terraform-parse-hcl (experimental)")
+	_ = cmd.Flags().MarkHidden("terraform-parse-hcl")
+	_ = cmd.Flags().MarkHidden("terraform-var-file")
+	_ = cmd.Flags().MarkHidden("terraform-var")
```

Comment:
when will users see these if they're hidden flags?

Reply:
They're not shown in the help text at all for the first cut.

Reply:
Is there anyway the user can see them at all then. How can they see the flag description if they're not part of `--help`. Is there a `--help --show-hidden`? 

So for the first draft we're essentially relying totally on the infracost docs to get people to find out about this right?

Reply:
> Is there anyway the user can see them at all then. 

Not currently.

For the first draft we're not making it very discoverable. We'll be reaching out to users directly to get them to try it and pointing them to the docs.

Reply:
👍